src/pyhf/infer/mle.py

```diff
@@ -0,0 +1,30 @@
+"""
+Module for Maximum Likelihood Estimation
+"""
+from .. import get_backend
+
+
+def loglambdav(pars, data, pdf):
+    return -2 * pdf.logpdf(pars, data)
+
+
+def fit(data, pdf, init_pars=None, par_bounds=None, **kwargs):
+    _, opt = get_backend()
+    init_pars = init_pars or pdf.config.suggested_init()
+    par_bounds = par_bounds or pdf.config.suggested_bounds()
+    return opt.minimize(loglambdav, data, pdf, init_pars, par_bounds, **kwargs)
+
+
+def fixed_poi_fit(constrained_mu, data, pdf, init_pars=None, par_bounds=None, **kwargs):
```

Comment:
```suggestion
def fixed_poi_fit(poi_val, data, pdf, init_pars=None, par_bounds=None, poi_index=None **kwargs):
```

and add sth like

```
poi_index = poi_index or pdf.config.poi_index
```

Reply:
hm I don't agree. the current API only knows a POI as a proprty of the model config. Allowing this to be overridden doesn't feel right (i.e. while it is `m.config.suggested_init()`, it's `m.config.poi_index` not `m.config.suggested_poi_index`). I would leave this overriding ability to whenever we do the larger refactor of passing in methods